doc/man/restic-backup.1

```diff
@@ -27,7 +27,7 @@ given as the arguments.
 .PP
 \fB\-\-exclude\-caches\fP[=false]
     excludes cache directories that are marked with a CACHEDIR.TAG file. See 
-\[la]http://bford.info/cachedir/spec.html\[ra] for the Cache Directory Tagging Standard
+\[la]https://bford.info/cachedir/spec.html\[ra] for the Cache Directory Tagging Standard
 
 .PP
 \fB\-\-exclude\-file\fP=[]
```

Comment:
This file is generated automatically during the build, please revert changes to it.